libpromises/patches.c

```diff
@@ -193,6 +197,8 @@ bool LinkOrCopy(const char *from, const char *to, int sym)
         return false;
     }
 
+#elif __TERMUX__    /* link() not supported on ANDROID platform */
+    return File_Copy(from, to);
```

Comment:
Why not `CopyFile()`?

Reply:
Isn't CopyFile() only in windows API?

Reply:
I didn't check, I just saw a different function being called. :)